src/Docker.DotNet/Models/Events.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Docker.DotNet.Models
+{
+    public class EventMessage
```

Comment:
This will reflect the JSONError and JSONProgress structs too which is the full message as far as I can tell.